apps/avifenc.c

```diff
@@ -209,7 +209,7 @@ static void syntaxLong(void)
            AVIF_SPEED_FASTEST);
     printf("\n");
     printf("Advanced options:\n");
-    printf("    -j,--jobs J                       : Number of jobs (worker threads, default: 1. Use \"all\" to use all available cores)\n");
+    printf("    -j,--jobs J                       : Number of jobs (worker threads). Use \"all\" to use all available cores (default: all)\n");
```

Comment:
Maybe: "to potentially use"
Just to make sure the user understands the limitations

Reply:
Actually, maybe: "to potentially use as many cores as possible"
In case machines have more than 64 cores ...

Reply:
Done. Thanks for the suggestion.